app/controllers/auctions_controller.rb

```diff
@@ -32,6 +32,14 @@ def update
 
   private
 
+  def find_auction
+    if params[:token]
+      Auction.find_by(token: params[:token])
```

Comment:
`token_find`, `draft_find`?

Also, just realized that the `:auction_id` in the URL is irrelevant since we're finding by the token. Is that ok?